### PR TITLE
Logic comparing image passed with request

### PR DESCRIPTION
**Pull request by mdyzma**
Endpoint `verify` takes request with file and checks for file presence. If positive will extract face features and compare with features stored in data.FEATURES dictionary.

### CODE REVIEW

File: face/compare.py [hunk 1]
```diff
@@ -1 +1,30 @@
...
+        picture_to_store, num_jitters=jitters)
+    return face_encoding[0].tobytes()
+
+
+def compare_features(unknown_features, known_features, tolerance=0.7):
+    result = face_recognition.compare_faces(known_features,
+                                            unknown_features,
+                                            tolerance=tolerance)
+
+    return result
```
Comment (pixycode) [inline face/compare.py]:
```suggestion
    return face_recognition.compare_faces(known_features,
                                            unknown_features,
                                            tolerance=tolerance)
```

Comment (mdyzma) [reply face/compare.py]: readability counts in my opinion...

Comment (mmarcinkiewicz) [reply face/compare.py]: What is the point of making a wrapper over a function without any modification to its functionality? You just pass parameters.

Comment (mmarcinkiewicz) [reply face/compare.py]: Moreover, assuming you compare features of faces, you change the well named function `compare_faces` into a vague `compare_features`

File: face/compare.py [hunk 2]
```diff
@@ -1 +1,30 @@
...
+
+def verify_user(unknown, known_users):
+    for _, val in known_users.items():
+        known = val.get('face_features')
+        known = np.frombuffer(known)
+        try:
+            if compare_features(unknown, known):
+                return val
+            else:
+                return {'error': 'unknown user'}
```
Comment (mmarcinkiewicz) [inline face/compare.py]: Is that for some sort of API requirements?

File: face/data.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+
+FEATURES = {'hopf_bifurcation':
```
Comment (mmarcinkiewicz) [inline face/data.py]: put some dummy data here, real data should not be put into repo

File: face/compare.py (hunk 2 above)
Comment (mdyzma) [reply face/compare.py]: yes

File: face/data.py (hunk 3 above)
Comment (mdyzma) [reply face/data.py]: they are needed for MVP tests. Google api wont work on dummy data.

File: face/compare.py (hunk 1 above)
Comment (mdyzma) [reply face/compare.py]: Good point.

File: face/data.py (hunk 3 above)
Comment (mmarcinkiewicz) [reply face/data.py]: Maybe ask support to make a dummy slack user for tests?

Comment (mdyzma) [reply face/data.py]: we need NG google account in fact

File: face/compare.py [hunk 4]
```diff
@@ -1 +1,22 @@
...
+
+def verify_user(unknown, known_users, tolerance=0.7):
+    for _, val in known_users.items():
+        known = val.get('face_features')
+        known = np.frombuffer(known)
+        try:
+            if face_recognition.compare_faces(known, unknown, tolerance=tolerance)[0]:
+                return val
+            else:
+                return {'error': 'unknown user'}
```
Comment (pixycode) [inline face/compare.py]: Suggestion: You can move an error structure to a function like `error_message(message)`. Then in one place you can change a structure of an error response

Comment (mdyzma) [reply face/compare.py]: Good point. I'll make it new JIRA task.

File: face/data.py (hunk 3 above)
Comment (pixycode) [inline face/data.py]: Please, keep PEP-8 convention. Each set of values in dict should be separated by 4 spaces
```suggestion
FEATURES = {
    'hopf_bifurcation': {
        'id': 0,
        'name': 'Michał',
        'surname': 'Dyzma',
        'email': 'michal.dyzma@netguru.com',
        'face_features': b"\x00\x00\x00\xe0\x01\xb6\xc1\xbf\x00\x00\x00\x80yx\xbd?\x00\x00\x00\x00\x1e\x1b\xb8?\x00\x00\x00\xa0\t'\xa1\xbf\x00\x00\x00\xa0U\xb3\xc0\xbf\x00\x00\x00\x00N(\xb5\xbf\x00\x00\x00 h,\x98\xbf\x00\x00\x00@\x1ev\xc0\xbf\x00\x00\x00\xe0U\x0e\xc8?\x00\x00\x00\xa08K\xbc\xbf\x00\x00\x00\x00\x82b\xc6?\x00\x00\x00\x00\x1c\xd3\x85?\x00\x00\x00`TP\xc2\xbf\x00\x00\x00\xe0\x00\xfe\x96\xbf\x00\x00\x00\x00\xf2\xd7\xaa\xbf\x00\x00\x00@\xfd\xf0\xb3?\x00\x00\x00`\x99\xb8\xb5\xbf\x00\x00\x00\xc0\xab\x0c\xb0\xbf\x00\x00\x00\x00\x96K\xbc\xbf\x00\x00\x00@I\x1c\xab\xbf\x00\x00\x00\x80\xde\xaeZ\xbf\x00\x00\x00\xa0\xb7\xc6\xb7?\x00\x00\x00\xc0[hj\xbf\x00\x00\x00\xc0\xafZ\xae?\x00\x00\x00\xa0\x88\xcc\x8c\xbf\x00\x00\x00\x00d/\xd0\xbf\x00\x00\x00\x80\\:\xbb\xbf\x00\x00\x00\x80\x9e\x8b\xa8\xbf\x00\x00\x00\x00\x9c)\xba?\x00\x00\x00\xa0Q\x89\xa1\xbf\x00\x00\x00\x80 -\x89?\x00\x00\x00 \xf5\t\xa4?\x00\x00\x00\x80\xc71\xc3\xbf\x00\x00\x00\x00\x94b\xb8\xbf\x00\x00\x00\xe0r\xe9\x98?\x00\x00\x00@\xc4\x8e\xad?\x00\x00\x00\xc0,\x1c\xbd\xbf\x00\x00\x00\xc0D\xca\xa6\xbf\x00\x00\x00 #\xb9\xc4?\x00\x00\x00\xe0\xe8\xfd\x93\xbf\x00\x00\x00\xa0\x98\x0e\xbb\xbf\x00\x00\x00\x80\xaa\x1a\xc0\xbf\x00\x00\x00\x00\x12\xf4\x90?\x00\x00\x00`o\xdf\xd0?\x00\x00\x00@j+\xc2?\x00\x00\x00\x80]j\xb3?\x00\x00\x00\xe0\x05?\x99?\x00\x00\x00@\xbd\xf0\xc3\xbf\x00\x00\x00\x80vf\xc3?\x00\x00\x00\xe0p\x7f\xcc\xbf\x00\x00\x00\xa0\x06>\xb6?\x00\x00\x00\xa0j\xa2\xbe?\x00\x00\x00\xa0w\xc8\xc1?\x00\x00\x00\x00.\xa9\xae?\x00\x00\x00\x00V>\xc2?\x00\x00\x00\xa0\x0eo\xbb\xbf\x00\x00\x00\xe0\xc4\xf2\xb8?\x00\x00\x00@\x06\xd4\x94?\x00\x00\x00\xa0d\xad\xc8\xbf\x00\x00\x00@\x88\x80\xb5?\x00\x00\x00\xa0\xa3x\xb7?\x00\x00\x00\xe0\x14\x1e\xaf\xbf\x00\x00\x00\x00Jf\xb7\xbf\x00\x00\x00@\xeez\xb0\xbf\x00\x00\x00\x80^\x9a\xc8?\x00\x00\x00\xa0\xcb\xd2\xb6?\x00\x00\x00\x00\xb3P\x95\xbf\x00\x00\x00 \xaf\x19\xc1\xbf\x00\x00\x00\x00\x84\xa1\xc8?\x00\x00\x00`\xa3N\xc9\xbf\x00\x00\x00`\x17\x12\xb3\xbf\x00\x00\x00 \xfc\x8f\xc0?\x00\x00\x00`e\xc1\xa0\xbf\x00\x00\x00 D\xef\xc5\xbf\x00\x00\x00`\x82\xe2\xcd\xbf\x00\x00\x00`\x81K\x9c\xbf\x00\x00\x00\xc0\xd5\x15\xdd?\x00\x00\x00\xe0@\xdb\xcd?\x00\x00\x00\x00M\xb5\xc1\xbf\x00\x00\x00\xe0~rz?\x00\x00\x00 \x91F\xb2\xbf\x00\x00\x00\xc0\xc4\x89\xb2\xbf\x00\x00\x00@LS\xbb?\x00\x00\x00\x80\xa5\xac\x8f?\x00\x00\x00@ (\xc2\xbf\x00\x00\x00\x00\xb6\x9e\x8e\xbf\x00\x00\x00\xe0`7\xb8\xbf\x00\x00\x00\x80\x8e5\xb0?\x00\x00\x00@\x91\r\xca?\x00\x00\x00\x00\x13\xc3\xaa\xbf\x00\x00\x00\xe0rj\xad\xbf\x00\x00\x00\xe0I}\xc8?\x00\x00\x00`\x00Q\xb4?\x00\x00\x00\x00d\xb1q\xbf\x00\x00\x00 \xedk\x90?\x00\x00\x00\x00\xa0x\xa5\xbf\x00\x00\x00`\x9d\xc7\xa7\xbf\x00\x00\x00\xc0|~\x9e?\x00\x00\x00\xe0\xb0\xf9\xbc\xbf\x00\x00\x00@Tq\xa8\xbf\x00\x00\x00\xa0\x8b\xab\xc5?\x00\x00\x00\xa0aV\x97\xbf\x00\x00\x00\x00\xa6%\xa6?\x00\x00\x00@\xb1#\xaa?\x00\x00\x00\x80R\xf2\xc4\xbf\x00\x00\x00\x80\x930\xca?\x00\x00\x00`U\xe4\x9a?\x00\x00\x00\xa0b\xad\x8c\xbf\x00\x00\x00\xc0\x82\x8d\xa3?\x00\x00\x00`\x8d\n\xaf\xbf\x00\x00\x00\x80\xf7\xb8\xca\xbf\x00\x00\x00\x00\xc4\xde\x83\xbf\x00\x00\x00\xa0\xf2\xf7\xca?\x00\x00\x00\xa0\x05\xd6\xc5\xbf\x00\x00\x00@\xf5O\xc8?\x00\x00\x00@\xddR\xaa?\x00\x00\x00\x80\xbf\xac\x85?\x00\x00\x00 \x1a\x8b\xc2?\x00\x00\x00@\xe2}\xa6?\x00\x00\x00\x80\xbc\xc6\xb9?\x00\x00\x00 $5^?\x00\x00\x00\xe0n2\xbb\xbf\x00\x00\x00\xe0y\xaf\xaf\xbf\x00\x00\x00 u\xf0\xba\xbf\x00\x00\x00@\x98h\xb7?\x00\x00\x00\x80jr\xa9\xbf\x00\x00\x00\xc0\x80\xea\xb9?\x00\x00\x00\xe0\x8c\xce\x9a?"
    }
}
```

**Pull request merged**